<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgImagenDigitos</class>
 <widget class="QDialog" name="dlgImagenDigitos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformacion de una Imagen</string>
  </property>
  <widget class="QPushButton" name="btnProcesarImagen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Procesar Imagen</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblImagen">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDigitoInicio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digito Inicio</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtDigitoInicio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblDigitoFin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digito Fin</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtDigitoFin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="txtNroVeces">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblNroVeces">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nro Veces</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
